scripts/linters/pylint_extensions_test.py

```diff
@@ -2234,3 +2234,191 @@ def test_file_overview_at_end_of_file(self):
+                        a = 1 + 2
+                """)
+        node_no_newline_below_class_docstring.file = filename
+        node_no_newline_below_class_docstring.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_no_newline_below_class_docstring)
+
+        message = testutils.Message(
+            msg_id='newline-below-class-docstring',
+            line=4)
+
+        with self.checker_test_object.assertAddsMessages(message):
+            temp_file.close()
+
+    def test_excessive_newline_below_class_docstring(self):
+        node_excessive_newline_below_class_docstring = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(Class):
+                        \"\"\"This is a docstring.\"\"\"
+
+
+                        a = 1 + 2
+                """)
+        node_excessive_newline_below_class_docstring.file = filename
+        node_excessive_newline_below_class_docstring.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_excessive_newline_below_class_docstring)
+
+        message = testutils.Message(
+            msg_id='newline-below-class-docstring',
+            line=6)
+
+        with self.checker_test_object.assertAddsMessages(message):
+            temp_file.close()
+
+    def test_class_with_no_docstring(self):
+        node_class_with_no_docstring = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(Class):
+                        def func():
+                            \"\"\"This is a docstring.\"\"\"
+                            a = b + c
+                """)
+        node_class_with_no_docstring.file = filename
+        node_class_with_no_docstring.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_class_with_no_docstring)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
+
+    def test_class_with_no_body(self):
+        node_class_with_no_body = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(Class):
+                        \"\"\"This is a docstring.\"\"\"
+
+                    def func():
```

Comment:
Is this a correct python file, this will raise an error, isn't it?

Reply:
No, this will not raise any error, except what we want, because this will run on only one check.

Reply:
I mean you cannot compile this file, right? (By compile I mean it will fail on lexical analysis phase.) Have you tried running a python file with this code?

Reply:
I have taken this example from the codebase. There are some files having classes like this i.e. without a body.

Reply:
@Hudda -- please add an explicit pointer to a specific example and include it in a comment in the code so that other developers who have the same question as @DubeySandeep can understand the reasoning here, too. Do this for all other similar comments that you are not addressing. Thanks!

Reply:
@seanlip @DubeySandeep  See this link(https://github.com/oppia/oppia/blob/22cbfdef8d321d6798cc21036aa2324180f9aec3/core/controllers/base.py#L107) for example.

Reply:
I have also added the link in inline comment.